doctr/__main__.py

```diff
@@ -60,9 +60,8 @@ def get_parser():
     deploy_parser.add_argument('--built-docs', default=None,
         help="""Location of the built html documentation to be deployed to
         gh-pages. If not specified, Doctr will try to automatically detect build location""")
-    deploy_parser.add_argument('--gh-pages-docs', default='docs',
```

Comment:
I was thinking of that, but I'm not sure how to go about having a default argument not be required if a certain non-default option is passed.